common/rabbitmq/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ Rabbitmq CHANGELOG
 
 This file is used to list changes made in each version of the common chart rabbitmq.
 
+0.8.0
```

Comment:
That has to be `0.9.0` because of https://github.com/sapcc/helm-charts/pull/6760/files#diff-74f4dc751816c78602d39a57153f066cdc639f87d8747b99bef85d9663185efeR6

Reply:
Updated to 0.9.0